packages/app/src/systems/Account/components/BalanceWidget/BalanceWidget.tsx

```diff
@@ -87,8 +100,8 @@ export function BalanceWidget({
           <VisibilityButton
             aria-label={isHidden ? 'Show balance' : 'Hide balance'}
             isHidden={isHidden}
-            onHide={() => setIsHidden(true)}
-            onShow={() => setIsHidden(false)}
+            onHide={() => toggleBalanceVisibility(true)}
```

Comment:
The name of the function was better before, toggle would be better for a function that doesn't receive the state. But in this case, we explicitly set is visible or not.

```suggestion
            onHide={() => setBalanceVisibility(true)}
```